google/cloud/aiplatform/models.py

```diff
@@ -5020,7 +5020,7 @@ def upload(
             if not model_dir.exists():
                 raise ValueError(f"artifact_uri path does not exist: '{artifact_uri}'")
             PREBUILT_IMAGE_RE = "(us|europe|asia)-docker.pkg.dev/vertex-ai/prediction/"
-            if re.match(PREBUILT_IMAGE_RE, serving_container_image_uri):
+            if serving_container_image_uri and re.match(PREBUILT_IMAGE_RE, serving_container_image_uri):
```

Comment:
Hi @rragundez , could you please fix the linting issue 

<img width="1237" alt="image" src="https://github.com/user-attachments/assets/27785b17-036a-4498-aacd-e5e02be60a0e">

Reply:
Done